imgs/installer/license.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Is it mandatory.
There is already the licence file at the root of the repo.

Reply:
#fixed and removed